pkg/server/plugin/upstreamauthority/awspca/pca_test.go

```diff
@@ -475,8 +481,11 @@ func (as *PCAPluginSuite) generateCSR() ([]byte, *bytes.Buffer) {
 }
 
 func (as *PCAPluginSuite) TestPublishJWTKey() {
-	resp, err := as.plugin.PublishJWTKey(ctx, &upstreamauthority.PublishJWTKeyRequest{})
-	as.Require().Nil(resp, "no response expected")
+	stream, err := as.plugin.PublishJWTKey(ctx, &upstreamauthority.PublishJWTKeyRequest{})
+	as.Require().NoError(err)
+	as.Require().NotNil(stream)
 
+	resp, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns io.EOF.